lib/include/openamp/remoteproc.h

```diff
@@ -432,6 +432,16 @@ struct remoteproc_ops {
 					  metal_phys_addr_t da,
 					  void *va, size_t size,
 					  struct remoteproc_mem *buf);
+	/**
+	 * can_recursive
+	 *
+	 * Check current vdev can do recursive rx_callabck or not
+	 *
+	 * @rproc - pointer to remoteproc instance
+	 *
+	 * return true if can do recursive rx_callabck.
+	 */
+	int (*can_recursive)(struct remoteproc *rproc);
 };
```

Comment:
I don't like this name. It doesn't immediately tell the reader what is going on. Something like rx_callback_can_recurse() is more explanatory, but still not good.